app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -1,16 +1,21 @@
 <template>
   <div>
     <v-row class="mt-6" no-gutters>
-      <v-col cols="12" sm="6">
+      <!-- page title -->
+      <v-col cols="12" md="6" order-sm="1">
         <h1>Form Design</h1>
+      </v-col>
+      <!-- form name -->
+      <v-col cols="12" order-md="3" order-sm="2">
         <h3 v-if="name">{{ name }}</h3>
       </v-col>
       <v-spacer />
-      <v-col class="text-sm-right" cols="12" sm="6">
+      <!-- buttons -->
+      <v-col class="text-md-right text-sm-left" cols="12" md="6" order="4" order-md="2" order-sm="4">
```

Comment:
Does this need to be split up into mutliple columns with responsive orders?
Can it just be 
| form design,  name, version (vertical)|-----------------------| buttons |
?

Reply:
Ohhh no is this so the form title can run **under** the buttons if it's long? In that case this probably works.
Though I'm not sure on line 14 if you need both `order="4"` AND `order-sm="4"`? Can the `sm` one be removed because the basic 4 one runs from xs to md?

Reply:
thats what i thought.. but i found you needed both wen testing

Reply:
That's weird, the spec says they're default and then breakpoint and **greater**

![image](https://user-images.githubusercontent.com/17445138/123173631-bfba2480-d433-11eb-949a-470566366155.png)

Reply:
If we do it that way, the mobile ordering gets messed up. That's why we have to structure it in this fashion.